src/LndClient/Orphan.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module LndClient.Orphan where
+
+import qualified Data.Text.Internal as T
+import LndClient.Class
+import LndClient.Import.External
+
+instance FromGrpc Text T.Text where
+  fromGrpc = Right . fromStrict
+
+instance ToGrpc Text T.Text where
+  toGrpc = Right . toStrict
```

Comment:
It seems like proto-lens is using strict data? Then we don't need this file probably, we can use strict data in our own models